### ISSUE TITLE
Allow usage of arrows to select data in WMTS viewer

### ISSUE THREAD

**Issue by clausmichele** [Open-EO/openeo-web-editor#278]
I find the WMTS viewer really useful if I don't know exactly if there is data covering the area I'm interested in. However, it is quite difficult to navigate in time, since I can just use the mouse to change the selected date. Would it be possible to allow the usage of keyboard arrows to do +- 1 day in the selected date?

![image](https://user-images.githubusercontent.com/31700619/190584613-554f5a06-28e6-412a-86cc-2efcb912f789.png)

**Comment by m-mohr** [comment]
Yeah, that time slider is really painful to use but it's the only one available right now. The idea with arrows is nice, but I think it would need to be implemented upstream in ol-ext: https://github.com/Viglino/ol-ext
The alternative would be to use another component for date selection. I've recently made it easier to add other components to the map, so we could also think about a date selector like in forms or so.

**Comment by christophfriedrich** [comment]
I don't know those components, but if it's the `Timeline` class, I've seen that it does have a `setDate` function ([docs](http://viglino.github.io/ol-ext/doc/doc-pages/ol.control.Timeline.html#setDate__anchor)), so it should be possible for the Editor to listen for keystrokes and issue function calls accordingly :thinking:

**Comment by m-mohr** [comment]
PRs are welcome :-P
